### ISSUE TITLE
Bluetooth LE GATT Client and Server not working on Hololens

### ISSUE THREAD

**Issue by AbhijithNB** [Microsoft/Windows-universal-samples#915]
Whenever I try to run the C# UWP app on Hololens, the app crashes with - 

> Unhandled exception at 0x6BBF8918 (Windows.UI.Xaml.dll) in BluetoothLEUniversal.exe: 0xC000027B: An application-internal exception has occurred (parameters: 0x0A7F8108, 0x00000003)

I have followed [This](https://social.msdn.microsoft.com/Forums/sqlserver/en-US/2303eddd-f3ad-4dad-b08c-93aad1321bb7/uwpunhandled-exception-applicationinternal-exception-in-windowsuixamldll?forum=wpdevelop). It did not help my case. Im also getting this exception - 
> Exception thrown: 'System.ArgumentOutOfRangeException' in BluetoothLEUniversal.McgInterop.dll
The operation attempted to access data outside the valid range (Exception from HRESULT: 0x8000000B)

I have also created another C# UWP application which just lists all the active Bluetooth LE Devices. It works fine and Im able to get the list of all active Bluetooth LE Devices and also pair with them.
Device Watcher code I have used - 
```
string[] requestedProperties = { "System.Devices.Aep.DeviceAddress", "System.Devices.Aep.IsConnected" };

string aqsAllBluetoothLEDevices = "(System.Devices.Aep.ProtocolId:=\"{bb7bb05e-5972-42b5-94fc-76eaa7084d49}\")";

DeviceWatcher deviceWatcher =
    DeviceInformation.CreateWatcher(
        aqsAllBluetoothLEDevices,
        requestedProperties,
        DeviceInformationKind.AssociationEndpoint);

// Register event handlers before starting the watcher.
deviceWatcher.Added += DeviceWatcher_Added;
//deviceWatcher.Updated += DeviceWatcher_Updated;
deviceWatcher.Removed += DeviceWatcher_Removed;
deviceWatcher.EnumerationCompleted += DeviceWatcher_EnumerationCompleted;
deviceWatcher.Stopped += DeviceWatcher_Stopped;

// Start over with an empty collection.
deviceList.Clear();

// Start the watcher.
deviceWatcher.Start();
```

And, Yes Im using x86 Platform Target.
BTW, this application works fine on Raspberry Pi 3 which runs Windows 10 IoT Core, but not on Hololens.
Any help is appreciated.

**Comment by oldnewthing** [comment]
I've asked the feature team to investigate. But it sounds like a platform issue, not a sample issue.

**Comment by oldnewthing** [comment]
The Bluetooth LE sample requires build 15063. HoloLens is still on 14393. This is a bug in the sample; it needs to set its minimum build to 15063. Fix coming in the next refrseh.

**Comment by oldnewthing** [comment]
Fixed in [v6.0.5](https://github.com/Microsoft/Windows-universal-samples/releases/tag/v6.0.5).
